.github/workflows/ckb-vm-test-suite.yml

```diff
@@ -15,11 +15,31 @@ jobs:
     - uses: actions/checkout@v4
       with:
         submodules: recursive
-    - name: Setup
+    - name: Setup CKB VM
+      run: |
+        cd ckb-vm-test-suite
+        make ckb-vm
+    - name: Setup Clang
+      run: |
+        clang --version
+        sudo ln -s $(which llvm-ar-18) /usr/bin/llvm-ar || true
```

Comment:
Why do we need those `ln` links? I think we can simply leverage `find_clang` then substitude the `clang` part with other binary name.

Reply:
Fixed, now use `find_clang` script.